scrapy/utils/conf.py

```diff
@@ -103,6 +108,18 @@ def init_env(project: str = "default", set_syspath: bool = True) -> None:
 
 def get_config(use_closest: bool = True) -> ConfigParser:
```

Comment:
I don't think this is the only function that needs changing, there are several others that look for `scrapy.cfg`.